examples/provider/bid/audience-network/src/audience-network-bid-delegate.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @file Audience Network BidDelegate.
+ */
+import 'whatwg-fetch';
+import isArray from 'lodash/isArray';
+import isFunction from 'lodash/isFunction';
+import isObject from 'lodash/isObject';
+import isString from 'lodash/isString';
+
+const name = 'audience-network';
+
+/**
+ * Does this slot contain valid parameters?
+ * @param {Object} slot
+ * @returns {Boolean}
+ */
+const validateSlot = slot =>
+  isObject(slot) &&
+  isString(slot.name) &&
+  isString(slot.elementId) &&
+  isArray(slot.sizes) &&
+  slot.sizes.length > 0 &&
+  isFunction(slot.getParam) &&
+  isObject(slot.getParam(name)) &&
+  isString(slot.getParam(name).placementId);
+
+/**
+ * Does this slot contain valid sizes?
+ * @param {Object} slot
+ * @returns {Boolean}
+ */
+const validateSlotSizes = slot => {
+  slot.sizes = slot.sizes.map(flattenSize);
+  return slot.sizes.every( size =>
+    ['native', 'fullwidth', '300x250', '320x50'].includes(size) );
```

Comment:
Agree - we saw the same issue as soon as people started using the Prebid.js equivalent, and I think it was switched to a filter instead. We should bid on the first size we recognise.